modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,379 @@
+    partition INT,
+    timestamp TIMESTAMP,
+    offset    LONG,
+    headers   ARRAY(OBJECT(KEY BINARY, VALUE BINARY NULLABLE)),
+    key       BINARY NULLABLE
+  ),
+   user_id INT,
+   event_type STRING NULLABLE,
+   ts STRING NULLABLE
+);
+
+```
+
+With schema integration, Redpanda uses the schema ID prefix embedded in each record to find the matching schema in the Schema Registry. Producers to the topic must use the schema ID prefix in the serialization process so Redpanda can determine the schema used for each record, parse the record according to that schema, and use the schema for the Iceberg table as well.
+
+=== Schema types translation
+
+[tabs]
+======
+Avro::
++
+--
+* Redpanda supports direct translations of the following Avro types to Iceberg value domains:
+** boolean
+** int
+** long
+** float
+** double
+** bytes 
+** string
+** record
+** array
+** maps
+** fixed
+** decimal
+** uuid
+** date
+** time
+** timestamp
+* Different flavors of timestamp and time types are all represented by the same Iceberg type, conversion will be done.
+* Avro unions are flattened to structs with optional fields, out of which only one is present, or none in case there is a field with null type:
+** For example, the union `["int", "long", "float"]` is represented as an Iceberg struct `OBJECT(0 INT NULLABLE, 1 LONG  NULLABLE, 2 FLOAT NULLABLE)`.
+** The union `["int", null, "float"]` is represented as an Iceberg struct `OBJECT(0 INT NULLABLE, 1 FLOAT NULLABLE)`.
+* All fields are required by default (Avro always sets a default in binary representation).
+* Duration logical type is ignored.
+* Null type is ignored and not represented in Iceberg schema.
+* Recursive types are not supported.
+--
+
+Protobuf::
++
+--
+* Redpanda supports direct translations of the following Avro types to Iceberg value domains: 
+** BOOL
+** DOUBLE
+** FLOAT
+** INT32
+** SINT32
+** INT64
+** SINT64
+** SFIXED32
+** SFIXED64
+** STRING
+** BYTES
+** MAP
+* Repeated values are translated into Iceberg ARRAY types.
+* Enums are translated into Iceberg INT types based on the integer value of the enumerated type.
+* UINT32 and FIXED32 are translated into Iceberg LONG types as that is the existing semantic for unsigned 32bit values in Iceberg.
+* UINT64 and FIXED64 values are translated into their base 10 string representation.
+* Timestamp well known protos are translated into TIMESTAMP Iceberg types.
+* Messages are converted into STRUCT types.
+* Recursive types are not supported.
+--
+======
+
+== Set up catalog integration
+
+You can configure the Iceberg integration to either create a file in the same object storage bucket or container to serve as the catalog, or connect to a REST-based catalog. 
+
+Set the cluster configuration property `iceberg_catalog_type` with one of the following values:
+
+* `object_storage`: Write catalog files to the same object storage bucket as the data files. Use the object storage URL to access the catalog for your Redpanda Iceberg tables.
```

Comment:
I think we should mention here it is not intended for production uses cases (at least for now)